examples/shakti-sdk_gpio-keypad/platformio.ini

```diff
@@ -15,11 +15,20 @@ description =
 platform = shakti
 framework = shakti-sdk
 board = artix7_35t
-monitor_speed = 115200
+monitor_speed = 19200
+debug_tool = ftdi
+upload_protocol = custom
+upload_port = = /dev/ttyUSB*
+
+
```

Comment:
Why do you need these fields?
```
debug_tool = ftdi
upload_protocol = custom
upload_port = = /dev/ttyUSB*
```
Also, the upload port shouldn't be specified here. What if a user has another device name? What about Windows users?

Reply:
Currently everything is tested on linux only. So for now linux, later in a while will look into windows .

Reply:
@valeros upload_protocol = custom is needed as it overrides the default ftdi configuration. Without this openocd won’t work .

Reply:
What do you mean by "won't work"?  Can you open a new issue and describe the error you see when uploading?